biosys/apps/main/utils_data_package.py

```diff
@@ -1335,3 +1335,15 @@ def to_workbook(self):
         ws = wb.create_sheet()
         self._to_worksheet(ws)
         return wb
+
+    def to_csv(self, output):
+        # TODO: remove when python3
+        if six.PY2:
+            import unicodecsv as csv
+        else:
+            import csv
```

Comment:
This import within a method is intentional. This class will be moved elsewhere soon.